internal/pkg/service/common/etcdop/iterator/iterator_test.go

```diff
@@ -222,6 +222,46 @@ ETCD_REQUEST[%d] ✔️️  GET ["some/prefix/foo005", "some/prefix0") | rev: %d
 	}
 }
 
+func TestIterator_AllKeys(t *testing.T) {
+	t.Parallel()
+
+	var logs strings.Builder
+	ctx := context.Background()
+	client := etcdhelper.ClientForTest(t)
+	client.KV = etcdlogger.KVLogWrapper(client.KV, &logs)
+	ops := []iterator.Option{iterator.WithPageSize(3)}
+
+	// Generate keys
+	prefix := etcdop.Prefix("") // <<<<< all!
```

Comment:
V migracii potrebujem preiterovat vsetky etcd kluce, ale to nas iterator nevedel.